Florida Tech Request (Reupload)

JacobChesslo opened the pull request:
For the request we have three prompt Dark Matter models, where they can decay into SM particles or first into DM matter particles and then into SM particles such as muons, through a spin 1 mediator, a dark Z-like boson ZD, which can interact with the Standard Model sector as well as the dark matter sector.

**FD model: DM_KM_Med_Spin1_Fermioic_Prds.:**
pp collisions produce our ZD through kinetic mixing, which then eventually decays into ordinary matter and stable dark matter (fD2)

**SD model: DM_KM_Med_Spin1_Scalar_Prds.**
pp collisions produce our ZD through kinetic mixing then decaying into dark scalar matter (sD), which then decays into ordinary matter

**Combination of the two models: DM_KM_Med_Spin1_Fermionic_Scalar_Prds**
A combination of the above two, where the dark Z can decay into either fermionic or scalar matter.

**Mass Scans**

MZD scan- fD model
MZD scan- sD model
MZD scan- combination of two models - fD decays
MZD scan- combination of two models - sD decays
MfD1 scan- combination of two models - fD decays
MfD2 scan- combination of two models - fD decays
Note: MsD scan- combination of two models - sD decays: we are not doing mass scan for MsD because they dont seem to have enough sensitivity for higher masses

This is a 2018 Request, and we would also like to clarify that **our sample campaign should be produced in consistency with ALP/Dark Susy 2018 samples**, such as in Wei's campaign: https://cms-pdmv.cern.ch/mcm/requests?dataset_name=ALP_mH_*_mALP_*_PSWeights_TuneCP5_13TeV-madgraph-pythia8&page=-1&shown=275146342527

_This is a Reupload of the original (Request #2666) where we were told to remove the "-pythia8" naming convention, as well as alteration to run_card with correct cuts._

agrohsje commented:
With whom did you exchange info? Why is the PDF changed to nnpdf 2.3? Which cuts exactly changed? It is a waste of our time to go through 30 files again. If only one or two cuts changed, there is no need for me to check the rest. Apparently you modified other things as well, which would force me to check it again. Please prepare this correctly and ping when ready.

JacobChesslo commented:
_With whom did you exchange info?_
-We exhanged info with Si Hyun Jeon and Jose Ruiz via emails

_Why is the PDF changed to nnpdf 2.3?_
-We had a misunderstanding on our end, the PDF should be corrected again.

Which cuts exactly changed?
-Included `set small_width_treatment default` in line 62
-Replaced `10.0 = ptl` with `5.0 = ptl` in line 97
-Removed drll cut : replaced `0.4 = drll` with `-1 = drll` in line 114

We would also like to clarify that **our sample campaign should be produced in consistency with ALP/Dark Susy 2018 samples**, such as in Wei's campaign: https://cms-pdmv.cern.ch/mcm/requests?dataset_name=ALP_mH_*_mALP_*_PSWeights_TuneCP5_13TeV-madgraph-pythia8&page=-1&shown=275146342527

JacobChesslo commented:
@agrohsje

agrohsje commented on bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZDfD_ZDM80toM700/ZDfD/ZDfD_run_card.dat:
```diff
@@ -0,0 +1,164 @@
+  0 = time_of_flight ! threshold (in mm) below which the invariant livetime is not written (-1 means not written)
+  average =  event_norm       ! average/sum. Normalization of the weight in the LHEF
+# To see MLM/CKKW  merging options: type "update MLM" or "update CKKW"
+  set small_width_treatment default
```
Did you test this syntax? I would expect the set command to work on the command line, while the run card should have something like value = variable. Please confirm.

agrohsje commented:
Any feedback on my comment?

JacobChesslo commented:
@agrohsje Please push through if everything checks out

JacobChesslo commented:
Is there a way to include [these three project-generator UFO](https://github.com/JacobChesslo/Research/tree/master/ZDarkModels) files in http://cms-project-generators.web.cern.ch/cms-project-generators/ In order to move forward with creating the gridpacks?

File Source: https://github.com/JacobChesslo/Research/tree/master/ZDarkModels

agrohsje commented:
Should work. Please check.
